src/index.html

```diff
@@ -1,12 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="index.css">
-    <title>Capstone DnD</title>
+    <title>Capstone</title>
 </head>
+
 <body>
-    This is a test!
+    <header class="container">
+        <nav>
+            <ul>
+                <li id="pokecounter">Pokemon</li>
```

Comment:
- Hey, please, kindly check the length well as your total item is not getting the correct length. you have 15 Pokemon, but it returns 12, on another reload, it returns 10.

Reply:
Hi! We tried to get the PokemonAPI to be faster, but it prints slowly and we have given it a set timeout to count all the pokemon. That's why it sometimes prints 10 or 12. Would you like us expand the timeout so that it always loads 15 or do you feel the change is not needed